README.md

```diff
@@ -1,14 +1,20 @@
 # GeoFabrics
 
+## Build Status
+
+![Linux](https://github.com/rosepearson/GeoFabrics/actions/workflows/python-test-package.yml/badge.svg)
+
+## Introduction
+
 The GeoFabrics and associated sub-packages include routines and classes for combining point (i.e. LiDAR), vector (i.e. catchment of interest, infrastructure) and raster (i.e. reference DEM) to generate a hydrologically conditioned raster. 
 
-GeoFabrics also contains support for downloading all LiDAR tiles within a spectified polygon (catchment region) from OpenTopography. This functionality is contained in the lidar_fetch module, and an example of its usage can be found in tests/test_lidar_fetch.
+GeoFabrics also contains support for downloading all LiDAR tiles within a spectified polygon (catchment region) from OpenTopography. This functionality is contained in the lidar_fetch module, and can be used independently of the other geofabrics modules. An example of its usage can be found in tests/test_lidar_fetch.
 
 ## import into a conda environment
 You can use this package by using pip to install the package and dependencies using the following commands (say in a conda terminal to add it to that environment), where the environment.yml is from the root of this repository.
 
 ```python
-conda env create -f environment.yml
+conda env create -f environment_[windows|linux].yml
```

Comment:
I like the use of this but it might pay to add a comment or use `$BASH_VARIABLES`, whichever you think is the best way to prevent users from thinking the command is literally `conda env create -f environment_[windows|linux].yml` instead of `conda env create -f environment_windows.yml`

Reply:
Good suggestion I'll have a quick go at clarifying it!

Reply:
ok I've had a stab at making it clearer in the text above. Let me know what you think.

Reply:
Looks great, just one quick suggestion then ready to go